### ISSUE TITLE
Display training and validation loss 

### ISSUE THREAD

**Issue by emtropyml** [IndicoDataSolutions/finetune#340]
While training there's no config setting to display the training/validation loss. Also, eval_acc does not work with multi-label classification. Also, autosave_path does not work. And many of the config settings are not updated in the official documentation. Would be good if you could add these. Thanks!

**Comment by benleetownsend** [comment]
The way to view losses and other training metrics is typically through tensorboard. However, if you set debugging_logs=True you will see the full estimator log output instead of the progress bars. There is a lot of noise here but it includes the train and validation losses. 

We will attempt to clean up the documentation for the config before our next release, thanks for pointing out autosave_path, this has not worked for quite some time

You are correct, eval_acc only works with single label classification. Adding a wide variety of metrics to track for every task type is not something we are likely to support. Accuracy on multilabel classification, for example, has a number of different interpretations. For each of these interpretations, the usefulness depends depends largely on what you want to use the model for. As seen in the examples in `finetune/datasets` the typical way we expect this to be done is to eval with a task appropriate metric after training on a held out dev/test set.

Thanks for your interest in finetune.
